src/app.js

```diff
@@ -1 +1,65 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+
+async function fetchAllCountries() {
+    try {
+        const result = await axios.get('https://restcountries.com/v2/all?fields=name,region,flag,population');
+        const countries = result.data;
+        // console.log(countries)
+        test(countries);
+
+    } catch (e) {
+        const country = document.getElementById("countriesUl");
+        country.innerHTML =
+            ` <li id="error">No countries found :(</li>
+`;
+        console.error(e);
+    }
+}
+
+fetchAllCountries()
+
+function test(countries) {
+    const countryElement = document.getElementById("countriesUl")
+    countries.sort((a, b) => a.population - b.population);
+    countryElement.innerHTML = countries.map((countries) => {
```

Comment:
Hier heb je een conflicteren naam, omdat `countries` al in gebruik is
```suggestion
    countryElement.innerHTML = countries.map((country) => {
```

En als je het nog verder wil trekken, zou je hier nog spreading kunnen toepassen zodat je niet overal `country.` vóór hoeft te zetten
```suggestion
    countryElement.innerHTML = countries.map(({ region, flag, name, population }) => {
```